src/changelog.rs

```diff
@@ -387,6 +387,30 @@ impl Changelog {
 
         self
     }
+
+    /// Add a link to the list of links
+    ///
+    /// # Examples
+    /// ```
+    /// # fn main() {
+    /// let mut changelog = ChangelogBuilder::default().build().unwrap();
+    ///
+    /// changelog.add_link("[anchor]: https://example.com".to_string());
+    ///
+    /// // Assert that the link was added correctly
+    /// assert_eq!(changelog.links().len(), 1);
+    /// assert_eq!(changelog.links().first().unwrap().anchor(), "anchor");
+    /// assert_eq!(changelog.links().first().unwrap().url(),"https://example.com");
+    /// # }    
+    ///     
+    pub fn add_link<T: Into<String>>(&mut self, link: T) -> &mut Self {
```

Comment:
mb better to make anchor and link as a separate arguments?

Reply:
Updated to collect strings for anchor and url with basic validation to strip brackets and colon if supplied on the anchor and ensure that the strings are not empty.